torch/distributed/fsdp/wrap.py

```diff
@@ -293,9 +293,9 @@ class HandleInitMode(Enum):
     This determines how :class:`FlatParameter` s are initially constructed for
     the parameter execution order policy. This construction is temporary, and
     the :class:`FlatParameter` s are reconstructed later.
-    MODULE_LEVEL: For each module, its immediately owned parameters are used to
-                  construct one :class:`FlatParameter`. This construction
-                  matches that of ``always_wrap_policy``.
+    MODULE_LEVEL: For each module or a list of modules, its immediately owned parameters
+                  are used to construct one :class:`FlatParameter`. In ``ParamExecOrderPolicy``,
+                  This construction is based on ``ParamExecOrderPolicy.module_level_group_policy``.
```

Comment:
nit: We should remember to make this consistent with `ParamExecOrderWrapPolicy` if we go with that name.